### DISCUSSION TITLE
Typescript Intigration

### DISCUSSION THREAD

**Discussion by anuragarwalkar** [hagopj13/node-express-boilerplate#74] [Ideas]
I would love to have typescript integrated with this template

**Comment by khkmalki** [comment]
I believe there are many other frameworks for backend typescript like
Nestjs and loopback.

The stability of this boilerplate is keeping work unified and easy to fix.

:) Of course you can fork and do great things with it.



On Sat, Feb 13, 2021 at 10:17 AM Anurag Arwalkar <notifications@github.com>
wrote:

> I would love to have typescript integrated with this template
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/hagopj13/node-express-boilerplate/discussions/74>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACSKKRMTEKMTAJGHF2UR67DS6YRSNANCNFSM4XR333MA>
> .
>
-- 
*Khalil Malki* *|*
*Sr. IT Developer | *
*404.394.8341 |*
*Delta  Airlines * *|*

**Comment by hagopj13** [comment]
@anuragarwalkar thanks for the suggestion. I'm actually planning on doing another boilerplate using TypeScript, and might as well do a few things differently there. As @khkmalki mentioned, [NestJS](https://nestjs.com/) is a really good option.

**Reply by samkit5495** [reply]
@hagopj13 did you work on this and have some boilerplate already ready
